composer/utils/eval_client/mcli_eval_client.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""MCLI compatible eval client."""
+import logging
+import os
+from typing import Dict, List
+import time
+import numpy as np
+
+import mcli
+
+from composer.utils.eval_client.eval_client import EvalClient
+
+__all__ = ['MCLIEvalClient']
+log = logging.getLogger(__name__)
+
+
+class MCLIEvalClient(EvalClient):
+    """Utility for creating a client for and invoking an AWS Lambda through MCLI."""
```

Comment:
I would suggest a rebrand here, similar to MosaicMLLogger. Would also make this specific to lambdas - we may want a more generic MosaicMLEvalClient in the future
```suggestion
class MosaicMLLambdaEvalClient(EvalClient):
    """Utility for creating a client for and invoking an AWS Lambda hosted on the MosaicML platform"""
```